frcxx_nft/src/types.rs

```diff
@@ -55,7 +56,57 @@ pub trait FRCXXXNFT {
     fn is_approved_for_all(&self, params: IsApprovedForAllParams) -> bool;
 }
 
-#[derive(Serialize_tuple, Deserialize_tuple, Debug)]
+/// Return value after a successful mint
+/// The mint method is not standardised, so this is merely a useful library-level type, and
+/// recommendation for token implementations
+#[derive(Serialize_tuple, Deserialize_tuple, Clone, Debug)]
+pub struct MintReturn {
+    /// The new balance of the owner address
+    pub balance: u64,
+    /// The new total supply
+    pub supply: u64,
+    /// List of the tokens that were minted successfully (some may have been burned during hook execution)
```

Comment:
Yes this makes sense for minting. It's trickier for transfers.

Reply:
https://github.com/helix-onchain/filecoin/issues/158